### ISSUE TITLE
Verify card dialog component customization

### ISSUE THREAD

**Issue by GaeCuure** [Adyen/adyen-ios#2162]
Hello, is there a way to customize this element? 
<img width="288" alt="Screenshot 2024-06-27 at 16 07 33" src="https://github.com/Adyen/adyen-ios/assets/131974978/7293ad62-b60d-4d95-b8b6-3a9c4a9ecec2">

**Comment by descorp** [comment]
Hey Gae,

what would you like to customize?

**Comment by GaeCuure** [comment]
I would like to customize the text font (title, body, placeholder, button title) and the button color

**Comment by descorp** [comment]
We have means of [UI customisation](https://github.com/Adyen/adyen-react-native/blob/develop/docs/Customization.md) and [Localisation customisation](https://github.com/Adyen/adyen-react-native/blob/develop/docs/Localization.md#overriding-default-values).

**Comment by GaeCuure** [comment]
yes, I have already followed the UI Customisation doc and was able to customize the dropIn but not the alert dialog.

**Comment by descorp** [comment]
Sorry for misleading you, @GaeCuure 

Indeed, there is no way to alter the appearance of [StoredCardAlert](https://github.com/Adyen/adyen-ios/blob/develop/AdyenCard/Components/Stored%20Card/StoredCardAlertManager.swift), only localized text.
We will discuss this idea with the product team. Thanks for feedback 💚
